### PR TITLE
ADD URL decode in header values in custom notification

### PR DESCRIPTION
**Pull request by fgalan**
Issue #4071

### CODE REVIEW

File: doc/manuals/user/forbidden_characters.md [hunk 1]
```diff
@@ -57,15 +57,17 @@ NGSIv2 introduces syntax restrictions for ID fields (such as entity id/type, att
 or metadata name/type) which are described in the "Field syntax restrictions" section in the
 [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable).
 
-## Custom payload special treatment
+## Custom payload and heaers special treatment
```
Comment (AlvaroVega) [inline doc/manuals/user/forbidden_characters.md:60]:
```suggestion
## Custom payload and headers special treatment
```

Comment (fgalan) [reply doc/manuals/user/forbidden_characters.md]: Fixed in [f75b7c6](https://github.com/telefonicaid/fiware-orion/pull/4072/commits/f75b7c6c374a09407eec72b867f7f377a0f0ead3)

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR does some modification in .md documents. It would be great if you could synchronize in the Japanese translation, please. Thanks!

**Comment by fisuda** [comment]
I send the PR #4074.
Thanks.

**Pull request merged**